server/helpers/tokenUtils.js

```diff
@@ -0,0 +1,44 @@
+import dotenv from 'dotenv';
```

Comment:
This entire tokenUtils file already exists in the middleware folder. I don't see why we have to duplicate it again here.

Reply:
Oops! that was a mistake. Nice spot, thanks.